crates/swc_ecma_transforms_typescript/src/transform.rs

```diff
@@ -1101,7 +1101,7 @@ impl Transform {
                                 expr.make_assign_to(
                                     op!("="),
                                     member_expr!(
-                                        DUMMY_SP.apply_mark(self.unresolved_mark),
+                                        DUMMY_SP.apply_mark(self.top_level_mark),
                                         module.exports
```

Comment:
It does look like `unresolved_mark` needs to be used here instead of `top_level_mark`, I'll confirm that later.

Reply:
However, `TsImportExportAssignConfig::Classic` is not used in any part of the code, so it may be unnecessary to handle it here.